flutter/cpp/c/backend_c.h

```diff
@@ -35,7 +35,8 @@ extern "C" {
 //         The app will load the next backend in the backend list.
 bool mlperf_backend_matches_hardware(const char** not_allowed_message,
                                      const char** settings,
-                                     const mlperf_device_info_t* device_info);
+                                     const mlperf_device_info_t* device_info,
+                                     const char* native_lib_path);
```

Comment:
Does it make sense to add `native_lib_path` as a new field to `mlperf_device_info_t` struct? This way we will not need to update the function `mlperf_backend_matches_hardware()` in other backends.